keycloak/openid_client_scope.go

```diff
@@ -13,6 +13,8 @@ type OpenidClientScope struct {
 	Attributes  struct {
 		DisplayOnConsentScreen string `json:"display.on.consent.screen"` // boolean in string form
 		ConsentScreenText      string `json:"consent.screen.text"`
+		GuiOrder               string `json:"gui.order"`
+		IncludeInTokenScope    string `json:"include.in.token.scope"` // boolean in string form
```

Comment:
You can set this attribute's type to `KeycloakBoolQuoted`, which is just a bool that understands how to marshal and unmarshal as a string.  Then you won't have to worry about converting it back and forth between those two types.

Reply:
OK, I'll use `KeycloakBoolQuoted`.